app/views/applications/show.html.slim

```diff
@@ -1,3 +1,3 @@
 / include webpacked react-table CSS
 == stylesheet_pack_tag 'react-table'
-== react_component 'ApplicationPage', { application: @application, fields: @fields, file_base_url: @file_base_url }
+== react_component 'ApplicationPage', { application: @application, fields: @fields, file_base_url: @file_base_url, fromPage: params[:from] }
```

Comment:
looks like file_base_url is snake_case and fromPage is not. we should be consistent about our field naming conventions